windows/src/desktop/kmshell/install/Keyman.Configuration.System.TIPMaintenance.pas

```diff
@@ -191,6 +228,8 @@ class function TTIPMaintenance.DoInstall(const KeyboardID,
     begin
       // The keyboard was not found
       KL.Log('lang not found: BCP47Tag = %s, CanonicalTag = %s', [BCP47Tag, CanonicalTag]);
+      // Additonal Sentry logging added to track TIP crash git hub issue #6619
+      TKeymanSentryClient.Client.MessageEvent(Sentry.Client.SENTRY_LEVEL_INFO,Format('lang not found: BCP47Tag = %s, CanonicalTag = %s', [BCP47Tag, CanonicalTag]),TRUE);
```

Comment:
I can't remember if in the end you said I didn't need to check if KeymanSentryClient.Client exsits?

Reply:
We should check, yes. Also adding `TTIPMaintenance.DoInstall: ` to the front of each message to make it easier to find them in log reports.
```suggestion
      if TKeymanSentryClient.Client <> nil then
        TKeymanSentryClient.Client.MessageEvent(Sentry.Client.SENTRY_LEVEL_INFO,
          Format('TTIPMaintenance.DoInstall: lang not found: BCP47Tag = %s, CanonicalTag = %s', [BCP47Tag, CanonicalTag]),
          TRUE);
```